metric/metric_test.go

```diff
@@ -112,7 +112,7 @@ func TestAddFieldOverwrites(t *testing.T) {
 
 	value, ok := m.GetField("value")
 	require.True(t, ok)
-	require.Equal(t, 42.0, value)
+	require.InDelta(t, 42.0, value, 0.001)
```

Comment:
I didn't use const here because of circular dependency to/from testutil package.